go/chat/attachments/uploader.go

```diff
@@ -331,20 +344,45 @@ func (u *Uploader) doneUploading(outboxID chat1.OutboxID) {
 	delete(u.uploads, outboxID.String())
 }
 
-func (u *Uploader) uploadFile(ctx context.Context, dirname, prefix string) (f *os.File, err error) {
+func (u *Uploader) getBaseDir() string {
+	u.Lock()
```

Comment:
Why add this?

Reply:
just to cover `tempDir` state with the lock